src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  --inpt-bckgr-color: #fff;
+  --inpt-brdr-color: #e1e7ed;
+  --inpt-text-color: #3d4e61;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+  font-size: 16px;
+}
+
+.image {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.image--small {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  padding-left: 62px;
+  line-height: 22px;
```

Comment:
It' better to use absolute values for line-height, like %

Reply:
oh, mg) thank`s for the tips!